en/releases/centreon-commercial-extensions.md

```diff
@@ -123,6 +123,13 @@ To use it, it is necessary to install and activate it. Refer to the
 
 ## Centreon Auto Discovery
 
+### 20.10.3
+
+#### Security fixes
+
```

Comment:
Missing mention to those fixes : 
[HD/Provider/Install/Upgrade] Limit provider definition version that is imported to 2.0 in 20.10/20.04
[Core] Creation forms generate 400 errors